app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -15,6 +15,8 @@ import { useDidUpdateEffect } from '@18f/identity-react-hooks';
 import { useI18n } from '@18f/identity-react-i18n';
```

Comment:
I followed the testing plan on the PR and everything worked! 🎉 And displayed conditionally as expected.

It looks like `service-provider-spec.jsx` has two failing tests due to the change in 0a66cee. It looks like removing `selfieCaptureEnabled` from line 15, and removing the tests / expectations on lines 20 and all of 35 - 42 will fix it. Or moving them to `feature-flag-spec.jsx` (which doesn't seem to exist yet).

Otherwise the other CI failures are still due to the missing translations.

Reply:
Thanks for going through the testing steps, I really appreciate that! Makes it much easier to merge confidently.

As you suggested, I also extracted the tests into a new file ([commit](https://github.com/18F/identity-idp/pull/9580/commits/9eeab883690303b08893f5970ba328bc399d58cd)). That file should be there anyway, and this made the tests clearer.